css-contain-3/Overview.bs

```diff
@@ -477,7 +477,7 @@ Naming Query Containers: the 'container-name' property</h3>
 		Initial: none
 		Inherited: no
 		Applies to: all elements
-		Computed Value: a set of unique [=identifiers=]
```

Comment:
No, this should be identifiers or `<custom-ident>` or whatever; ident sequence described a sequence of characters, not a CSS value.